server/index.js

```diff
@@ -191,10 +196,22 @@ Visit the documentation for more information: https://bpanel.org/docs/configurat
     level: 'info'
   });
   await blgr.open();
+
+  // setting up whitelisted ports for wsproxy
+  // can add other custom ones via `proxy-ports` config option
+  const ports = [18444, 28333, 28901].concat(
```

Comment:
could you add a comment saying which numbers correspond to which networks?

Reply:
these were actually just in the default browser implementation on bcoin. I'm not entirely sure what these corresponded to, but for the network specific ones I just use the Network module